core/src/test/scala/org/broadinstitute/dsde/rawls/monitor/MultiregionalBucketMigrationActorSpec.scala

```diff
@@ -911,4 +911,256 @@ class MultiregionalBucketMigrationActorSpec extends AnyFlatSpecLike with Matcher
+        }
+
+        job <- startBucketTransferJob(migration, testData.workspace, GcsBucketName("foo"), GcsBucketName("bar"))
+        finished <- nowTimestamp
+        _ <- inTransaction { _ =>
+          storageTransferJobs
+            .filter(_.jobName === job.getName)
+            .map(_.finished)
+            .update(finished.some)
+        }
+
+        job <- peekTransferJob.mapF(optionT => OptionT(optionT.value.map(_.some)))
+      } yield job should not be defined
+    }
+
+  "refreshTransferJobs" should "update the state of storage transfer jobs" in
+    runMigrationTest {
+      for {
+        migration <- inTransactionT { dataAccess =>
+          OptionT.liftF(createAndScheduleWorkspace(testData.workspace)) *>
+            dataAccess.multiregionalBucketMigrationQuery.getAttempt(testData.workspace.workspaceIdAsUUID)
+        }
+
+        _ <- startBucketTransferJob(migration, testData.workspace, GcsBucketName("foo"), GcsBucketName("bar"))
+        transferJob <- refreshTransferJobs
+      } yield {
+        transferJob.migrationId shouldBe migration.id
+        transferJob.finished shouldBe defined
+        transferJob.outcome.value shouldBe Outcome.Success
+      }
+    }
+
+  it should "update the state of jobs in order of last updated" in
+    runMigrationTest {
+      val storageTransferService = new MockStorageTransferService {
+        // want to return no operations to make sure that the job does not complete and is
+        // updated continually
+        override def getTransferJob(jobName: JobName, project: GoogleProject) =
+          IO.pure {
+            TransferJob.newBuilder
+              .setName(jobName.value)
+              .setProjectId(project.value)
+              .setStatus(TransferJob.Status.ENABLED)
+              .build
+          }
+      }
+
+      MigrateAction.local(_.copy(storageTransferService = storageTransferService)) {
+        for {
+          migration1 <- inTransactionT { dataAccess =>
+            OptionT.liftF(createAndScheduleWorkspace(testData.workspace)) *>
+              dataAccess.multiregionalBucketMigrationQuery.getAttempt(testData.workspace.workspaceIdAsUUID)
+          }
+
+          migration2 <- inTransactionT { dataAccess =>
+            OptionT.liftF(createAndScheduleWorkspace(testData.workspace2)) *>
+              dataAccess.multiregionalBucketMigrationQuery.getAttempt(testData.workspace2.workspaceIdAsUUID)
+          }
+
+          _ <- startBucketTransferJob(migration1, testData.workspace, GcsBucketName("foo"), GcsBucketName("bar"))
+          _ <- startBucketTransferJob(migration2, testData.workspace2, GcsBucketName("foo"), GcsBucketName("bar"))
+
+          getTransferJobs = inTransaction { _ =>
+            storageTransferJobs
+              .sortBy(_.id.asc)
+              .result
+              .map(_.toList)
+          }
+
+          transferJobsBefore <- getTransferJobs
+
+          _ = Thread.sleep(500)
```

Comment:
Without this, the test was flaking because `transferJobsMid(0).updated` and `transferJobsBefore(0).updated` were often equal. Adding this sleep has made the test pass consistently. There is another test that I had to do this for that was flaking in the last PR. 

I am sadly not sure why this is happening with these tests, but doesn't appear to be an issue with the PPW tests

Reply:
Just double checked the PPW tables, and the `updated` column goes up to 6 digits of precision while the multiregional bucket table doesn't. It only really matters for these tests, but I'll look into adding some additional precision to deflake the tests over these sleeps